DDL/StoredProcedures/create_student_request_table.sql

```diff
@@ -2,8 +2,15 @@ CREATE PROCEDURE CreateStudentRequestTable
 AS
 CREATE TABLE StudentRequests(
     RequestID INT PRIMARY KEY,
-    StudentID INT FOREIGN KEY REFERENCES,
+    StudentID INT FOREIGN KEY REFERENCES Students(StudentID),
     Motivation VARCHAR(MAX),
     Amount MONEY,
-    RequestStatus VARCHAR(20)
-)
\ No newline at end of file
+    RequestStatus VARCHAR(20),
```

Comment:
Replace RequestStatus with StatusID as a Foreign key that refrences the statuses table status ID column

Reply:
im already doing that on line 9 or am i missing something?

Reply:
yeah you can remove line 8 since you have the ID on line 9